assets/src/application/services/subscription/useSubscriptionService.ts

```diff
@@ -33,6 +25,21 @@ const useSubscriptionService: SubscriptionServiceHook = ({ domain, service }) =>
 		return window[NAMESPACE]?.[domain]?.[service]?.subscriptions || {};
```

Comment:
```suggestion
		return getServiceRegistryItem('subscriptions, {}');
```

only works if a default param is added to that function though

Reply:
This actually triggers some TS errors which would complicate the things further, that's why I left it as such